src/ast/converter.js

```diff
@@ -709,7 +709,7 @@ export default class Converter {
   }
 
   escapeLikePercent(value) {
-    return value.replace(/\%/g, '\\%').replace(/_/g, '\\_%');
+    return value.replace(/\\/g, '\\\\').replace(/\%/g, '\\%').replace(/_/g, '\\_%');
```

Comment:
In theory, I agree.  But I also suspect that somebody (probably us) are exploiting this somehow.